CMakeLists.txt

```diff
@@ -0,0 +1,218 @@
+#
+# CMakeLists.txt
+#
+# This source file is part of the FoundationDB open source project
+#
+# Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+cmake_minimum_required(VERSION 3.5)
+project(fdb
+  VERSION 6.1.0
+  DESCRIPTION "FoundationDB is a scalable, fault-tolerant, ordered key-value store with full ACID transactions."
+  HOMEPAGE_URL "http://www.foundationdb.org/"
```

Comment:
`DESCRIPTION` and `HOMEPAGE_URL` are added in CMake 3.9 and 3.12, respectively. Can you bump the `cmake_minimum_required` to 3.12, please? As it is, build breaks if you have CMake older than 3.12.